psql2mysql.py

```diff
@@ -236,10 +236,12 @@ def do_migration(config):
                           title="Commands",
                           help="Available commands",
                           handler=add_subcommands),
-        cfg.StrOpt('source',
+        cfg.URIOpt('source',
+                   schemes = [ "postgresql", "mysql", "mysql+pymsql"],
```

Comment:
Does it make sense to have anything else than  "postgresql" here?

Reply:
Yeah, while it might be easy to adapt the tool for other source databases, some querys against  the source database are postgresql specific. E.g. the one defined in `_query_utf8mb4_rows`.